commands/slash/resume.js

```diff
@@ -25,7 +25,7 @@ const command = new SlashCommand()
         embeds: [
           new MessageEmbed()
             .setColor("RED")
-            .setDescription("There is no song playing right now."),
+            .setDescription("You must be in a voice channel to use this command!"),
```

Comment:
I'm pretty sure this message is incorrect. It checks for player being undefined not for being inside a voice channel

Reply:
There is no song playing right now would also be wrong since the player needs to be created, in a voice channel, with a user to play a song

Reply:
> There is no song playing right now would also be wrong since the player needs to be created, in a voice channel, with a user to play a song

If the player is not initialized that means the bot is not inside a VC. This is totally unrelated to the user being in a channel or not. You have used a wrong explanation for this case.

Reply:
> There is no song playing right now would also be wrong since the player needs to be created, in a voice channel, with a user to play a song

I wrote that, and trust me it's the most sensical thing for that condition to return that message.
If you want to contest on it, please learn the `getChannel.js` functions and look at how the other commands are laid out. 

Since the player doesn't exist its obvious that it isn't playing music, and thus its obvious that it cant resume a song... because nothing is playing...